src/popup/pages/Networks.vue

```diff
@@ -32,6 +32,7 @@
 import { defineComponent } from 'vue';
 import { useDispatch, useGetter } from '../../composables/vuex';
 
+import { ROUTE_NETWORK_ADD } from '../router/routeNames';
```

Comment:
As ROUTE_NETWORK_ADD isn't a component I'd suggest to group it with other files
```suggestion
import { useDispatch, useGetter } from '../../composables/vuex';
import { ROUTE_NETWORK_ADD } from '../router/routeNames';

```